aviatrix/data_source_aviatrix_transit_gateway.go

```diff
@@ -256,6 +256,11 @@ func dataSourceAviatrixTransitGateway() *schema.Resource {
 				Computed:    true,
 				Description: "A map of tags assigned to the transit gateway.",
 			},
+			"enable_multitier_transit": {
```

Comment:
Should we add a underscore between multi and tier, like `enable_multi_tier_transit`? 

multitier can be hard to read if you don't know what it is supposed to be.